xcube_geodb/sql/geodb--1.0.0.dev0.sql

```diff
@@ -218,7 +218,7 @@ BEGIN
     RETURN QUERY EXECUTE format('SELECT JSON_AGG(src) as js ' ||
                                 'FROM (SELECT
                                              name as database,
-                                             regexp_replace(table_name, name || ''_'','''') as table_name,
+                                             regexp_replace(table_name, name || ''_'','''') as 'collection',
```

Comment:
Needs to be double quotes: 

```suggestion
                                             regexp_replace(table_name, name || ''_'','''') as "collection",
```